cmd/test/main.go

```
@@ -0,0 +1,111 @@
+		fmt.Printf("  - %s\n", key)
+	}
+
+	// Test 4: Delete operation
+	fmt.Println("\nTest 4: Delete operation")
+	err = bc.Delete("greeting")
+	if err != nil {
+		log.Printf("Failed to delete greeting: %v", err)
+	}
+
+	// Try to get deleted key
+	value, err = bc.Get("greeting")
```

golangci-lint (1.64.8)

ineffectual assignment to value

(ineffassign)